### ISSUE TITLE
how to get a webform from the web page

### ISSUE THREAD

**Issue by jomarocas** [signalpoint/webform#9]
is possible get a webform from the web, i have a webform create before, is possible get the form as a node

**Comment by signalpoint** [comment]
@jomarocas Yes, webforms are just nodes, so you can retrieve them as usual with a **GET** to `?q=drupalgap/node/123.json`, using `node_load(123, { success: function(node) {} });` or by just navigating to `node/123` in DrupalGap which will automatically render the form.
